### ISSUE TITLE
Setup: Console warning for deprecated color-adjust

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#861]
Color adjust seems to be deprecated. Locate the property and rename to `print-color-adjust`.

![image](https://user-images.githubusercontent.com/1659006/206460321-1156280f-cad6-42e7-a797-c2bc13c1124e.png)

```bash
./src/styles.scss.webpack[javascript/auto]!=!./node_modules/css-loader/dist/cjs.js??
ruleSet[1].rules[6].rules[0].oneOf[0].use[1]!./node_modules/postcss-loader/dist/cjs.js??
ruleSet[1].rules[6].rules[0].oneOf[0].use[2]!./node_modules/resolve-url-loader/index.js??
ruleSet[1].rules[6].rules[1].use[0]!./node_modules/sass-loader/dist/cjs.js??ruleSet[1].rules[6].rules[1].use[1]!./src/styles.scss - Warning: 
Module Warning (from ./node_modules/postcss-loader/dist/cjs.js):
Warning

(2699:3) autoprefixer: Replace color-adjust to print-color-adjust. The color-adjust shorthand is currently deprecated.
```

**Comment by gfellerph** [comment]
Might be fixed with #860

**Comment by oliverschuerch** [comment]
I will close this issue, because there is no such warning anymore and I could not found any `color-adjust` statement in our code.
